Safety/out-of-hours-support.rst

```diff
@@ -39,17 +39,19 @@ Once enrolled, you can `download`_ a desktop application or browser extension to
     Your number will also be accessible to everyone with permissions to the vault.
     If you volunteer, please provide your availability to the best of your knowledge, including holdiays and weekends.
```

Comment:
Typo in holidays. 
Also, you mention pingraham in the first paragraph as the person to request access to the Vault, shouldn't his name be removed?

Reply:
Yup, damn. I will put in A2, I think. 👍